### PR TITLE
Create newsletter-20200423.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
Adding April newsletter

### CODE REVIEW

**Comment by GeorgiaHCA** [comment]
Heya @malvikasharan - would you be able to review this for me? I'd really appreciate it!

File: project-communications/newsletter-20200423.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+breaks: False
+---
```
Comment (KirstieJane) [inline project-communications/newsletter-20200423.md:3]:
```suggestion
```

This is a hackmd command.

File: project-communications/newsletter-20200423.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+If you'd like to be interviewed, or to interview someone else, please get in touch with Georgia at gaitkenhead@turing.ac.uk
+
+This month, we're delighted to introduce Otis Smith. 
+
+Otis has been involved in multiple aspects of the project, including giving feedback on designs, presentations, and agendas; helping to design the labelling system for the focus group summaries; helping to prioritise tasks; giving ideas and insights in workshops; and co-authoring the blog written for World Autism Awareness day. 
+He also co-wrote this newsletter!  
+
+Here he is:
+
+![](https://i.imgur.com/c7d8kQ7.jpg)
```
Comment (KirstieJane) [inline project-communications/newsletter-20200423.md:101]: Can you add this picture to the repo please and link.

File: project-communications/newsletter-20200423.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+### Blog
+
+Otis Smith, who is an autistic collarborator on the citizen science project, and Georgia Aitkenhead, a Turing researcher, co-authored a blog to celebrate World Autism Awareness Day. You can read the blog here: https://www.turing.ac.uk/blog/world-autism-awareness-day-celebration-and-citizen-science
+
+### Workshop
+
+Wireframing workshop: On the 27th March we ran a wireframing workshop over Zoom. 
+The workshop was a mix of people from Autistica, Fujitsu, the Turing, and community members. 
+You can see the notes from the meeting by following this link: https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-wireframing-workshop/project-management/workshops/wireframing-March2020-notes.md
+
```
Comment (KirstieJane) [inline project-communications/newsletter-20200423.md:58]: I think you're missing an image here.

File: project-communications/newsletter-20200423.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+* Put your comments in our [always open google form](https://docs.google.com/forms/d/e/1FAIpQLSe_9MW3r4nsGUDxGWN24X88hqtz6q_347DC1gbMc_Yzj2Hrcg/viewform)
+* Get in touch with the team directly on [GitHub](https://github.com/alan-turing-institute/AutisticaCitizenScience) or [Gitter](https://gitter.im/alan-turing-institute/AutisticaCitizenScience).
+  You will need to create an account, but you don't have to provide your real name or any details.  
+* Email Georgia at [gaitkenhead@turing.ac.uk](mailto:gaitkenhead@turing.ac.uk)
+
+### Blog
+
+Otis Smith, who is an autistic collarborator on the citizen science project, and Georgia Aitkenhead, a Turing researcher, co-authored a blog to celebrate World Autism Awareness Day. You can read the blog here: https://www.turing.ac.uk/blog/world-autism-awareness-day-celebration-and-citizen-science
+
```
Comment (KirstieJane) [inline project-communications/newsletter-20200423.md:52]: Are you missing an image here?

**Comment by GeorgiaHCA** [comment]
Heya Kirstie! Thanks for your review! Sorry forgot about the images - I've added the images to the blog now and included them in the images folder in the project-communications folder: https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-communications/images

File: project-communications/newsletter-20200423.md (hunk 3 above)
Comment (GeorgiaHCA) [reply project-communications/newsletter-20200423.md:57]: Image added 👍

File: project-communications/newsletter-20200423.md (hunk 4 above)
Comment (GeorgiaHCA) [reply project-communications/newsletter-20200423.md:49]: Image added 👍

File: project-communications/newsletter-20200423.md (hunk 2 above)
Comment (GeorgiaHCA) [reply project-communications/newsletter-20200423.md:102]: Added to the repo here: https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-communications/images

**Pull request merged**